README.md

```diff
@@ -100,6 +101,41 @@ vrs_bulk ps
 
 The command line utility supports Google Cloud URIs and running commands in the background to interop with Terra out-of-the-box. This is described in the CLI usage above. For an example notebook, see `vrs-anvil-demo.ipynb` on the `vrs-anvil` workspace.
 
+## Cohort Allele Frequency Generation
+
+### Description
+Create a cohort allele frequency object for a given variant, subsettable by participant list, GREGOR-formatted phenotypes, etc.
```

Comment:
the expected entry point here is as a terra workflow, right (not an independent CLI)? Might be worth mentioning to distinguish it from the preceding text in the README

Reply:
Update it to explain it will be used as a Python package. See [example Terra notebook](https://app.terra.bio/#workspaces/WagnerLab/vrs_anvil/analysis/launch/caf-example.ipynb) if you haven't already

Reply:
Oh d'oh ok. Maybe a quickstart code snippet or something would be helpful, then?